ui/app/AppLayouts/Chat/controls/community/InlineNetworksComboBox.qml

```diff
@@ -109,7 +116,9 @@ StatusComboBox {
         title: root.control.displayText
         iconSource: instantiator.icon
 
-        amount: !d.oneItem ? instantiator.amount : ""
+        amount: !d.oneItem
+                ? (instantiator.infiniteAmount ? "∞" : instantiator.amount)
```

Comment:
Maybe we can add the `infinite` symbol as a readonly property in this component since it is used in different places.